kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/Tab8021xUi.java

```diff
@@ -367,6 +440,18 @@ public void setNetInterface(GwtNetInterfaceConfig config) {
 
         this.username.setValue(config.get8021xConfig().getUsername());
         this.password.setValue(config.get8021xConfig().getPassword());
+
+        if (this.activeConfig.getPassword().isEmpty()
+                || GwtSafeHtmlUtils.htmlUnescape(PASSWORD_PLACEHOLDER).isEmpty()) {
+            this.password.setValue(GwtSafeHtmlUtils.htmlUnescape(PASSWORD_PLACEHOLDER));
+            no8021xPasswordInSnapshot = true;
```

Comment:
I don't think we need this variable.

Reply:
I was doing this to try to circumvent the 'magic variable' problem. I will make this change.